airgun/views/rhsso_login.py

```diff
@@ -13,3 +13,13 @@ class LoginView(View, ClickableMixin):
     def is_displayed(self):
         return self.browser.wait_for_element(
             self.username, exception=False) is not None
+
+
+class RhssoExternalLogoutView(View, ClickableMixin):
+    login_again = Text('//a[@href="/users/extlogin"]')
+    logo = Text('//a[@alt="logo')
```

Comment:
Does that selector match anything? As far as I know, `alt` is not legal attribute of `a`. Perhaps you meant `img`?

Reply:
copy + paste